src/Compilers/VisualBasic/Test/Semantic/Diagnostics/IOperationTests.vb

```diff
@@ -57,63 +49,36 @@ End Module
             Assert.Equal(nodes.Length, 3)
 
             ' x = x + 10 fails semantic analysis and does not have an operator method, but the operands are available.
-
             Assert.Equal("x = x + 10", nodes(0).ToString())
-            Dim statement1 As IOperation = model.GetOperation(nodes(0))
-            Assert.Equal(statement1.Kind, OperationKind.ExpressionStatement)
-            Dim expression1 As IOperation = DirectCast(statement1, IExpressionStatement).Expression
-            Assert.Equal(expression1.Kind, OperationKind.AssignmentExpression)
-            Dim assignment1 As IAssignmentExpression = DirectCast(expression1, IAssignmentExpression)
-            Assert.Equal(assignment1.Value.Kind, OperationKind.BinaryOperatorExpression)
-            Dim add1 As IBinaryOperatorExpression = DirectCast(assignment1.Value, IBinaryOperatorExpression)
-            Assert.Equal(add1.BinaryOperationKind, BinaryOperationKind.OperatorMethodAdd)
-            Assert.False(add1.UsesOperatorMethod)
-            Assert.Null(add1.OperatorMethod)
-            Dim left1 As IOperation = add1.LeftOperand
-            Assert.Equal(left1.Kind, OperationKind.LocalReferenceExpression)
-            Assert.Equal(DirectCast(left1, ILocalReferenceExpression).Local.Name, "x")
-            Dim right1 As IOperation = add1.RightOperand
-            Assert.Equal(right1.Kind, OperationKind.LiteralExpression)
-            Dim literal1 As ILiteralExpression = DirectCast(right1, ILiteralExpression)
-            Assert.Equal(CInt(literal1.ConstantValue.Value), 10)
+            comp.VerifyOperationTree(nodes(0), expectedOperationTree:="
```

Comment:
Would it be OK to have both the old form of the verification and the new? I understand that the old form is technically redundant so long as the new form is working, but the tests are already written and would be faster to debug in the event of future failures.